externals/CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ endif()
 # MagicEnum
 if (NOT TARGET magic_enum::magic_enum)
     add_subdirectory(magic_enum)
+    target_include_directories(magic_enum INTERFACE magic_enum/include/magic_enum)
```

Comment:
Headers are now installed in `/usr/include/magic_enum/magic_enum.hpp`. You should edit all *.cpp files, replacing `#include <magic_enum.hpp>` with `#include <magic_enum/magic_enum.hpp>` instead.